### PR TITLE
Refactor SDK: remove unused file recognition

### PR DESCRIPTION
**Pull request by tmathern**
## Description

I noticed we had part of our library that does file-type recognition, but it was neither complete nor used internally in the sdk (or by drilling down in any worker). So this PR is to remove it from the SDK (it was an internal API anyway), remove the dependency it pulled in and the unit-tests.

Please do not pay attention to the branch name. I thought removing that was important enough to deserve its own PR...

Fixes # (issue)
Part of [NUI-607](https://jira.corp.adobe.com/browse/NUI-607)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Refactoring

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by tmathern** [comment]
https://github.com/adobe/asset-compute-sdk/pull/12

File: package.json [hunk 1]
```diff
@@ -60,6 +59,7 @@
   "scripts": {
     "test": "nyc mocha test --file test/logfile.setup.js --recursive --exit",
     "posttest": "eslint ./ && license-checker --summary && ./node_modules/lockfile-lint/bin/lockfile-lint.js --path package-lock.json —type npm --allowed-hosts npm artifactory.corp.adobe.com --allowed-schemes \"https:\" \"file:\" --empty-hostname true",
-    "beautify": "eslint ./ --fix"
+    "beautify": "eslint ./ --fix",
+    "unit-test": "nyc mocha test --file test/logfile.setup.js --recursive --exit"
```
Comment (alexkli) [inline package.json:63]: This is exactly the same as `test`?

Comment (tmathern) [reply package.json:63]: Nearly. There is no eslint or license check running after this one.

Comment (tmathern) [reply package.json:63]: (I'll rework those scripts in another PR anyway).

Comment (alexkli) [reply package.json:63]: Then to achieve DRY you could do:

```
"test": "npm run unit-test",
"unit-test": "nyc mocha ...."
```

**Comment by alexkli** [comment]
Note that `27.1.0` was an erroneous release. It was unpublished.

**Pull request merged**